mmv1/products/workstations/terraform.yaml

```diff
@@ -30,3 +34,79 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         primary_resource_id: "default"
         vars:
           workstation_cluster_name: "workstation-cluster-private"
+  WorkstationConfig: !ruby/object:Overrides::Terraform::ResourceOverride
+    import_format:
+      [
+        "projects/{{project}}/locations/{{location}}/workstationClusters/{{workstation_cluster_id}}/workstationConfigs/{{workstation_config_id}}",
+        "{{workstation_cluster_id}}",
```

Comment:
```suggestion
```
This line should be deleted from the array, as it causes the documentation to say you can import a workstation config with the id of a workstation cluster ([here's a preview of how the docs would be generated from the PR currently that shows this](https://github.com/modular-magician/terraform-provider-google-beta/blob/13b7d73767b3ff56fdc482ef6e295e23b24021c6/website/docs/r/workstations_workstation_config.html.markdown#import))

Reply:
Removed this line.